pkg/pkgmgmt/client/runner.go

```diff
@@ -34,7 +34,7 @@ func (r *Runner) Validate() error {
 		return errors.New("mixin not specified")
```

Comment:
Assuming the field name is updated, etc., we should update the error text to use the generic `package` as opposed to `mixin`...

Reply:
You're right, I'll do a passthrough on the entire pkgmgmt package for "mixin" and fix the text.